--- mxml-2.8/Makefile.in.orig	2014-04-19 10:26:41.746170424 +0200
+++ mxml-2.8/Makefile.in	2014-04-19 10:37:35.666156796 +0200
@@ -297,7 +297,7 @@
 
 libmxml.so.1.5:	$(LIBOBJS)
 	echo Creating $@...
-	$(DSO) $(DSOFLAGS) -o libmxml.so.1.5 $(LIBOBJS)
+	$(DSO) $(DSOFLAGS) -o libmxml.so.1.5 $(LIBOBJS) -lpthread
 	$(RM) libmxml.so libmxml.so.1
 	$(LN) libmxml.so.1.5 libmxml.so
 	$(LN) libmxml.so.1.5 libmxml.so.1
